com.github.sdbg.debug.ui/src/com/github/sdbg/ui/launchers/IChromeLaunchShortcutExt.java

```diff
@@ -0,0 +1,11 @@
+package com.github.sdbg.ui.launchers;
+
+import org.eclipse.core.resources.IResource;
+
+public interface IChromeLaunchShortcutExt {
```

Comment:
See above, let's rename this to just IChromeLaunch.

By the way: SDBG supports already several launch configurations (and their number will grow when we implement support for other browsers besides Chrome):
1. Chrome debug launch (the one you are targeting right now)
2. Chrome _app_/_extension_ debug launch
3. Chrome Mobile (Android) launch
4. Android port forwarding launch. Goes in tandem with the Chrome Mobile launch from above, and helps you to more easily access the website running locally on your PC to an Android phone attached via a USB cable
5. Attaching to a remote Chrome instance (limited applicability, but could be useful in some rare cases)

Any ideas how to expose support for 2, 3, 4 and 5 in addition to 1?